synadm/api.py

```diff
@@ -38,7 +38,6 @@ def query(self, method, urlpart, params=None, data=None):
                 url, headers=self.headers, timeout=self.timeout,
                 params=params, json=data
             )
-            resp.raise_for_status()
             if not resp.ok:
                 self.log.warning("No valid response from Synapse")
```

Comment:
Did you mean that I should reword this to "WARNING Synapse returned XXX:"? I assume the `resp.json()` output will be pretty-printed by some kind of outputter?

Reply:
exactley, the resp.json is taken care of already :-)

Reply:
Updated the warning. Tested, but manually by editing my other PR on `make-admin`.